corehq/feature_previews.py

```diff
@@ -127,3 +127,12 @@ def enable_callcenter(domain_name, checked):
     ),
     privilege=LOOKUP_TABLES,
 )
+
+
+SHARE_CASE_TILE_CONTEXT = FeaturePreview(
```

Comment:
Don't think this should be a feature preview since there are other toggles that are required to make this work.